client/pages/review-form.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import Navbar from '../components/navbar';
+export default class ReviewForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      country: '',
+      countries: []
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleCancelTripReview = this.handleCancelTripReview.bind(this);
+  }
+
+  componentDidMount() {
+    fetch('/api/countries', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    })
+      .then(response => response.json())
+      .then(result => {
+        this.setState({
+          countries: result
+        });
+      });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const token = window.localStorage.getItem('TravelApp-token');
+    let countryId = this.state.countries.findIndex(country => country.name === this.state.country);
+    countryId += 1;
+    const review = {
+      countryId,
+      country: this.state.country,
+      city: event.target.city.value,
+      review: event.target.review.value
+    };
```

Comment:
If you implement the rest of the changes suggested, you will be able to send the countryId, city, and review from your state object.